mne/channels/channels.py

```diff
@@ -88,46 +88,86 @@ def _get_ch_type(inst, ch_type, allow_ref_meg=False):
 
 
 @verbose
-def equalize_channels(candidates, verbose=None):
-    """Equalize channel picks for a collection of MNE-Python objects.
+def equalize_channels(instances, copy=False, verbose=None):
+    """Equalize channel picks and ordering across multiple MNE-Python objects.
+
+    First, all channels that are not common to each object are dropped. Then,
+    using the first object in the list as a template, the channels of each
```

Comment:
you change the current behavior by taking the first at template. BEfore it was taking the one with the highest number of channels so order did not matter

Reply:
That is something I do wish to change. By using the first object as template, the user can more easily control the order in which the channels end up in. For example, it would be computionally beneficial to specify the largest object in memory first, so those channels do not need to be re-ordered which would be expensive. Using the object that originally has the most channels seems to random to me. If there was a good reason for it, I'd love to hear it (@dengemann?)

Reply:
then it should at least be documented as an API change in what's new.